--- 
title: "Work"
---

> My goal is to contribute to the understanding of the driving biological mechanisms of diseases in order to develop new cures, with a special fascination for our immune system. I am particularly interested in combining existing public resources, mechanistic understanding of diseases, and high throughput/omics data using computational methods to a more complete picture of the underlying causes.

 
## Relevant experience 

*Incoming | October 2024* <br>
**Mann Lab** Max Planck Institute for Biochemistry | Munich <br>
PhD studies in the department for *Proteomics and Signal Transduction*. 

*May 2023 - July 2024* <br>
**Shalek Lab** Broad Institute | Ragon Institute of Harvard, MIT and MGH | IMES/MIT | Boston <br>
Metaanalysis of recurrent T cell states in health and disease. Development, validation, and application of a novel deep learning model for the integration of experimental side information with single-cell RNAseq data ([$\rightarrow$ scCoral](https://github.com/lucas-diedrich/sccoral.git)). 

*September 2022 - December 2023* <br>
**Alexandrov Lab** EMBL Heidelberg | Heidelberg <br>
Integrating immunocytochemistry with single cell metabolomics in primary human CD4^+^ T cells for drug screening. Experimental work, data analysis and computational work. 

*February 2022 - March 2023* <br>
**Graeter Lab** Heidelberg Institute for Theoretical Studies | Heidelberg <br>
Evaluation of the membrane permeability of small molecules with molecular dynamics simulations. Establishing a computational pipeline for data analysis 
([$\rightarrow$ Manuscript](https://doi.org/10.1002/adfm.202315731)).

*May 2022 - July 2023* <br>
**Russell Lab** Bioquant | Heidelberg <br>
Computational biology. Evaluating the pathogenicity of missense variants in protein domains using public resources. 

*March 2022 - October 2023* <br>
**Hartmann Group** Max Planck Institute for Biology | Tübingen <br>
Establishing a biophysical assay for the screening of natural and artificial ligands of the E3 ligase Cereblon. 


## Education 

*2021 - 2024* <br>
**Biochemistry M.Sc.** Ruprecht-Karls-University Heidelberg | Heidelberg <br>
Final Grade: 1.0 <br> 
Computational biology, biophysics, drug discovery 

*2018 - 2021* <br>
**Biochemistry B.Sc.** Eberhard Karls University Tübingen | Tuebingen <br>
Final Grade: 1.1 


## Social 

*2023* <br>
**Member/Organizer** German American Conference <br>
Functional team social events

*2022 - today* <br>
**IT-Team** Student Body Chemistry/Biochemistry, Heidelberg <br>
Member - Maintenance of website and IT infrastructure, email communication. 

*2020-2021* <br>
**Treasurer** Student Body Biochemistry, Tübingen <br>
Treasurer of for the student body, an organization with 300 members. 

*2021* <br>
**STEM representative** University of Tübingen <br>
Representing the subject biochemistry at local high schools to motivate young talents for a career in STEM. 

## Publications 

- L. Diedrich, M. Brosz, T. Abele, N. Urbanek, S. Steinke, F. Gräter, K. Göpfrich, C. Aponte-Santamaría, Energetics and Kinetics of Membrane Permeation of Photoresists for Bioprinting. Adv. Funct. Mater. 2024, 34, 2315731. [https://doi.org/10.1002/adfm.202315731](https://doi.org/10.1002/adfm.202315731)

## Software

### Languages 
**Python** (*advanced*)

**R** (*intermediate*)

**Bash** (*advanced*)


### Software packages

**ipathapy** Python | [github](https://github.com/lucas-diedrich/ipathapy.git)<br>
A python package to create metabolic path maps with ipath3 

**sccoral** Python | [github](https://github.com/lucas-diedrich/sccoral) | [docs](https://sccoral.readthedocs.io/en/latest/) <br>
A variational autoencoder with linear decoder for the inference of interpretable gene programs from single-cell data


## Scholarships 

*December 2020 - March 2024* <br>
**German Scholarship Foundation**

*April 2024 - July 2024*  <br>
**ERASMUS+**

*March 2020 - December 2020* <br>
**Deutschlandstipendium**


## Rewards 
*January 2025* <br>
**jGBM Master's thesis Reward** <br>
Best Master's thesis in Bioinformatics in the year 2024. 